README.md

```diff
@@ -1,2 +1,47 @@
 # comp130-journal
 Base repository for COMP130 research journal
+
+[] are used to denote the sourced paper with the number corresponding to the paper in the source list.
```

Comment:
Also you should apply this convention consistently -- for example in several places you refer to e.g. "paper 1" rather than "[1]"